scripts/helpers/utils.js

```diff
@@ -0,0 +1,59 @@
+const dockerNames = require('docker-names');
+const fs = require('fs');
+const { promisify } = require('util');
+const uuid = require('uuid/v4');
+
+const appendFileAsync = promisify(fs.appendFile);
+const writeFileAsync = promisify(fs.writeFile);
+
+const buildAppStoreUrl = ({ appId, tsApiHost, vendorId }) =>
+	`${tsApiHost.replace('api-', '')}/#/apps/Tradeshift.AppStore/apps/${vendorId}.${appId}`;
+
+const buildAppEmbedUrl = ({ appId, tsApiHost, vendorId }) =>
+	`${tsApiHost.replace('api-', '')}/#/apps/${vendorId}.${appId}`;
+
+const buildAppUrl = ({ appId, host, vendorId }) =>
+	`https://${vendorId.toLowerCase()}${appId.toLowerCase()}.${host}/`;
```

Comment:
Would the ever be a case where the protocol is not `https`?

Reply:
tradeshift requires iframed apps to be hosted on https...hence all of the ngrok/localtunnel stuff.